README.md

```diff
@@ -1 +1,580 @@
+| align            | string  | If the character is Good, Bad or Neutral                                                                                                     |
+| eye              | string  | Eye color of the character                                                                                                                   |
+| hair             | string  | Hair color of the character                                                                                                                  |
+| sex              | string  | Sex of the character (e.g. Male, Female, etc.)                                                                                               |
+| gsm              | string  | If the character is a gender or sexual minority (e.g. Homosexual characters, bisexual characters)                                            |
+| alive            | string  | If the character is alive or deceased                                                                                                        |
+| appearances      | int     | The number of appearances of the character in comic books <sup>*</sup> |
+| first appearance <sup>**</sup> | string  | The month and year of the character’s first appearance in a comic book, if available                                                         |
+| year           | int     | The year of the character’s first appearance in a comic book, if available                                                                   |
+
+<sup>\* as of Sep. 2, 2014. Number will become increasingly out of date as time goes on</sup>
+
+<sup>\** `first appearance` formatting can be different </sup>
+
+
+#### Example Output
+```json
+# Example DC Output:
+[
+  {
+    "page_id": 1422,
+    "name": "batman (bruce wayne)",
+    "urlslug": "/wiki/batman_(bruce_wayne)",
+    "id": "secret identity",
+    "align": "good characters",
+    "eye": "blue eyes",
+    "hair": "black hair",
+    "sex": "male characters",
+    "gsm": "",
+    "alive": "living characters",
+    "appearances": 3093,
+    "first appearance": "1939, may",
+    "year": 1939
+  }
+]
+
+# Example Marvel Output:
+[
+  {
+    "page_id": 1678,
+    "name": "spider-man (peter parker)",
+    "urlslug": "/spider-man_(peter_parker)",
+    "id": "secret identity",
+    "align": "good characters",
+    "eye": "hazel eyes",
+    "hair": "brown hair",
+    "sex": "male characters",
+    "gsm": "",
+    "alive": "living characters",
+    "appearances": 4043,
+    "first appearance": "aug-62",
+    "year": 1962
+  }
+]
+```
+
+## Service
+
+#### Requirements
+node.js (9+)
+pm2 `npm install pm2`
+
+#### Installation
+1. Clone the repo `git clone <REPO>`
+1. cd into the directory and install node.js requirements `npm install`
+1. Generate a self signed cert `openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -out sss.pem -keyout sss.pem -days 365`
```

Comment:
I had to manually split the .pem file into a ssh-cert and ssh-key files. Not sure if that was user error on my end or what

Reply:
Nope, I had the same issue and forgot to update the instructions.